src/libraries/Directory.Build.targets

```diff
@@ -128,6 +128,8 @@
   <Import Project="$(RepositoryEngineeringDir)slngen.targets" Condition="'$(IsSlnGen)' == 'true'" />
 
   <Import Project="$(RepositoryEngineeringDir)illink.targets" Condition="'$(IsSourceProject)' == 'true' or '$(ExplicitlyImportCustomILLinkTargets)' == 'true'" />
+  <!-- liveILLink.targets should be conditionally imported after illink.targets (which also imports liveILLink.targets) to prevent a double-import. -->
+  <Import Project="$(RepositoryEngineeringDir)liveILLink.targets" Condition="'$(_ImportedLiveILLinkTargets)' != 'true'" />
```

Comment:
Changes look really good except for these two lines (which I don't understand). illink.targets already imports liveILLink.targets. Why do we need to import it here explicitly?